sdb/commands/spl/internal/kmem_helpers.py

```diff
@@ -0,0 +1,155 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+from drgn.helpers.linux.list import list_for_each_entry
+
+from sdb.commands.linux.internal import slub_helpers as slub
+
+#
+# See spl/sys/kmem_cache.h
+#
+KMC_BITS = [
+    "KMC_BIT_NOTOUCH",
+    "KMC_BIT_NODEBUG",
+    "KMC_BIT_NOMAGAZINE",
+    "KMC_BIT_NOHASH",
+    "KMC_BIT_QCACHE",
+    "KMC_BIT_KMEM",
+    "KMC_BIT_VMEM",
+    "KMC_BIT_SLAB",
+    "KMC_BIT_OFFSLAB",
+    "KMC_BIT_DEADLOCKED",
+    "KMC_BIT_GROWING",
+    "KMC_BIT_REAPING",
+    "KMC_BIT_DESTROY",
+    "KMC_BIT_TOTAL",
+    "KMC_BIT_ALLOC",
+    "KMC_BIT_MAX",
+]
```

Comment:
Not a big deal, but could we figure this out from the type information in the target, perhaps after naming this enum in kmem_cache.h?

Reply:
Opened https://github.com/zfsonlinux/zfs/pull/9478
and updated the code in this PR